src/components/icons/day-calendar/DayCalendar.tsx

```diff
@@ -0,0 +1,21 @@
+import { DEFAULT_ICON_SIZE } from '../.././../models';
+import React from 'react';
+import { useTheme } from '../../../providers';
+import { Icon, SVGProps } from '../..';
```

Comment:
```suggestion
import { Icon, SVGProps } from '../../icon';
```
Please update all the imports to avoid circular dependencies issues

Reply:
@hiddeottens fixed for all icons now

Reply:
👍